security/certManager/file/certManager.go

```diff
@@ -72,23 +80,28 @@ func (a *CertManager) watchFiles() {
 	}
 }
 
-// NewFileCertManager creates a new certificate manager which watches for certs in a filesystem
-func NewFileCertManager(cas []*x509.Certificate, dirPath string, tlsKeyFileName string, tlsCertFileName string) (_ *CertManager, mgrErr error) {
-
+// NewFileCertManagerFromConfiguration creates a new certificate manager which watches for certs in a filesystem
+func NewFileCertManagerFromConfiguration(config Config) (*CertManager, error) {
```

Comment:
pls rename NewFileCertManagerFromConfiguration to NewCertManagerFromConfiguration